### DISCUSSION TITLE
Video "PyTorch for Deep Learning "-Having a command not found for "!nvidia-smi"

### DISCUSSION THREAD

**Discussion by rivhurvs7** [mrdbourke/pytorch-deep-learning#1057] [Q&A] [answered]
![colab](https://github.com/user-attachments/assets/b0874a5a-b876-49b2-9e94-c5b97b510d71)
please tell how can i fix "!nvidia-smi " command not found issue

**Comment by LuluW8071** [comment, marked as answer]
@rivhurvs7 You need to connect to a T4 GPU instance in order to access the GPU
Go to runtime menu option > change runtime type > **select T4 GPU**
![image](https://github.com/user-attachments/assets/e984efaf-5384-40ae-9fe1-7147f07a1dbf)

Then, re run the command
```bash
!nvidia-smi
```

**Reply by rivhurvs7** [reply]
@LuluW8071 thank you! its working now

**Comment by bmaxdk** [comment]
True when you're running a Jupyter notebook or a Python environment that does not have access to a GPU. The `!nvidia-smi` command is used to check the status of NVIDIA GPUs available on your system. If you don't have a GPU available, or if your current runtime is not connected to a GPU, this command will not be found.

As shown above, in hardware accelerator dropdown, select GPU (T4, K80, etc., depending on what's available for you).

**Reply by rivhurvs7** [reply]
@bmaxdk thank you!
